docs/team.rst

```diff
@@ -119,7 +119,7 @@ Responsibilities
 Members
 ```````
 
-* `Kelly O'Brien <https://twitter.com/OBrienEditorial>`_ *
+* `Beth Aitman <https://twitter.com/baitman>`_ *
```

Comment:
Maybe keep Kelly on the team for now, but add yourself with the `*`? Or maybe add a list of "past contributors" or some other way of acknowledging the work folks have done?

Reply:
```suggestion
* `Beth Aitman <https://twitter.com/baitman>`_ *
* `Kelly O'Brien <https://twitter.com/OBrienEditorial>`_ 
```

Reply:
Good shout, have done. I do love this new suggestions feature!